argusclient/client.py

```diff
@@ -391,7 +391,18 @@ def add(self, alert):
             alertobj.trigger = alertobj.triggers.add(alert.trigger)
         if alert.notification:
             alertobj.notification = alertobj.notifications.add(alert.notification)
-        if alert.trigger and alert.notification:
+        elif alert.notifications and isinstance(alert.notifications, list):
+            notifications = []
+            for notification in alert.notifications:
+                notifications.append(alertobj.notifications.add(notification))
+            alertobj.notifications = notifications
+        if alert.trigger and alert.notifications:
+            alertobj.trigger.notificationsIds = []
+            for notification in alertobj.notifications:
```

Comment:
This would change to `alertobj.notifications.items()`, as this would remain a service object at this point. I guess the difference in the type of this object for new vs. existing alert is going to make it confusing, but since the iteration is only useful on an existing alert, it may not be all that bad.

Reply:
As I mentioned earlier, `alertobj.notifications` should remain a service object, so this iteration will change.